arch/arm/src/tlsr82/tc32/tc32_doirq.c

```diff
@@ -102,7 +111,7 @@ static int locate_code(".ram_code") tc32_getirq(void)
  *
  ****************************************************************************/
 
-void arm_ack_irq(int irq)
+void locate_code(".ram_code") arm_ack_irq(int irq)
```

Comment:
can we switch all the places to use `__ramfunc__`?

Reply:
All the codes have been fully tested, i do not recommend to modify this.

Reply:
Yes. Sure. This is a note for the further development. There is already section in RAM for functions, so maybe the existing section can be renamed to match common approach

Reply:
I see. Then it makes sense to keep it as is